<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingComplexityDialog</class>
 <widget class="QDialog" name="TrainingComplexityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Training Complexity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="complexityGroupBox">
     <property name="title">
      <string>Complexity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="easyRadioButton">
        <property name="text">
         <string>&amp;Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mediumRadioButton">
        <property name="text">
         <string>&amp;Medium</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hardRadioButton">
        <property name="text">
         <string>&amp;Hard</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="translationCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Translation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
